pipelines/tekton/aiedge-e2e/tasks/model-registry-download.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: model-registry-download
+spec:
+  description: This Task can be used to fetch a model properties from model registry when enabled or properties passed by pipelinerun 
+  params:
+  - name: model-name
+    type: string
+  - name: model-version
+    type: string
+  #Values to be passed by pipeline run when model-register is disabled
+  - name: s3-bucket-name
+    type: string
+    default: ""
+  - name: modelRelativePath
+    type: string
+  - name: model-dir
+    type: string
+    default: "."
+  - name: fetch-model
+    type: string
+  - name: git-model-repo
+    type: string
+    default: ""
+  - name: git-model-revision
+    type: string
+    default: "main"
+  - name: model-registry-hostname
+    type: string
+    default: ""
+  - name: model-registry-enabled
+    type: string
+  results:
+  - name: fetch-model
+    type: string
+    description: The fetch mechanism that will be forwarded to the pipeline
+  - name: git-model-repo
+    description: The GIT URI that will be forwarded to the pipeline
+  - name: s3-bucket-name
+    description: The s3 Bucket name that will be forwarded to the pipeline
+  - name: modelRelativePath
+    type: string
+  - name: model-dir
+    type: string
+  - name: git-model-revision
+    type: string
+  steps:
+  - name: download-model-registry
+    image: registry.access.redhat.com/ubi9/ubi
+    script: |
+      #!/usr/bin/env bash
+
+      if [ "$(params.model-registry-enabled)" == "true" ]; then
```

Comment:
So what is the difference between model-registry-enabled being true or false? Are the fetch tasks (git, s3) supposed to work either way?

Reply:
@jackdelahunt  Thanks for the comment. I updated the Readme to reflect the answer to this. Hope that helps.